src/app/static/src/app/components/plots/FilterSet.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-
     <div class="form-group" v-for="f of filters" :key="f.stateFilterId">
-        <label class="font-weight-bold">{{f.label}}</label>
-        <!-- For some reason using filter is saying no component registered
-        with this name, no idea why so using capital Filter -->
-        <Filter :state-filter-id="f.stateFilterId" :plot="plot"/>
+        <div v-if="f.visible">
+            <label class="font-weight-bold">{{f.label}}</label>
+            <!-- For some reason using filter is saying no component registered
+            with this name, no idea why so using capital Filter -->
```

Comment:
i believe this happens when vue gets confused with the name, `filter` might be a special name for vue, i know in vue 2 there was a `Vue.filter` so it might be confusing it with that special filter, happy to keep it `Filter` i think that works, if you do want to stick with consistent casing, you can call it something else
```ts
import FilterComponent from "./Filter.vue";
```
and use it as `<filter-component/>` and you should not get the error, but up to you!